app/views/subscriptions/update.html.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :page_title_prefix, 'Subscription updated'
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    .govuk-panel.govuk-panel--confirmation
+      %h1.govuk-panel__title= t('subscriptions.update.header')
+      .govuk-panel__body
+        = t('subscriptions.reference')
+        %br
+        %strong= @subscription.reference
+
+    %h3.govuk-heading-m=t('subscriptions.confirmation.next_step')
```

Comment:
Can we duplicate the content into a new key `subscriptions.update.next_step`? I'm nervous about confirmation page content edits accidentally changing this content unintentionally.

Reply:
Good shout 👍